util-core/src/main/scala/com/twitter/util/Batcher.scala

```diff
@@ -0,0 +1,20 @@
+package com.twitter.util
+
+/** Provides a clean, lightweight interface for working with batched Futures */
+private[util] class Batcher[In, Out](
```

Comment:
OK thanks will do.  Scala syntax often has me confused about where the physical delineation is in the code between the class and its constructor :)